app/src/main/java/pl/edu/pjwstk/s999844/shoppinglist/adapters/ShoppingListAdapter.kt

```diff
@@ -52,15 +52,17 @@ class ShoppingListAdapter(private val changeAmountCallback: BiConsumer<RequiredI
 		binding.amountTextView.isVisible = item.amount > 1
 
 		binding.nameTextView.text = item.name
+		binding.nameTextView.alpha = if (item.amount > 0) 1f else 0.3f
```

Comment:
The alpha value should be a constant value since it's used more than once.
```kotlin
companion object {
    private const val ZERO_COUNT_OPACITY = 0.3f
}
```